### ISSUE TITLE
Can't log in even after uninstalled wakatime

### ISSUE THREAD

**Issue by Ranoiaetep** [wakatime/xcode-wakatime#52]
It's sort of continued from [Issue #49](https://github.com/wakatime/xcode-wakatime/issues/49), and on Xcode 12.3 here.

Even after uninstalling wakatime, the issue still won't fix. It would still say "Couldn't communicate with a helper application". Seems like the easiest way to solve it might be to reinstall Xcode.

Seems like the problem was my keychain and certificate files for Xcode were deleted during installation. There was one point my certificate files are moved to home folder, but they were deleted after the installation. 

Probably due to the same reason, my Github account on Xcode can't stay logged in. It requires me to reenter my token every time I quit and reopen Xcode.

It might be a good idea to suggest clone your Xcode as the default installation method on the [installation page](https://wakatime.com/xcode)

**Comment by qyzdo** [comment]
I have the same problem, every time I reinstall Xcode it runs fine, and when I install wakatime plugin after few days I have "Couldn't communicate with a helper application". It's really annoying, hope It will be fixed soon because currently, wakatime for me is almost unusable because of this.

**Comment by MohammadHossein-1996** [comment]
if it's possible for you please fix this bug as soon as you can.
I want to know have many hours I work in a day with wakaTime I can plan my day but now.
thanks.

**Comment by megatunger** [comment]
Please fix this bug...

**Comment by stockerb** [comment]
I agree with the others. Please fix this issue. I love this plugin for helping me analyze my work. However, when it comes to releasing software to the app store, it prevents me access to my Apple account for signing and verification.

**Comment by alanhamlett** [comment]
Does keeping two versions of Xcode (one with WakaTime and one without for publishing) help? We've been trying to find a way to fix this, but it doesn't seem possible without Apple's help currently.

**Comment by david-gettins** [comment]
> Currently we don't have a way to fix it without help from Apple.

Have you tried requesting code level support via your developer acount? Think you need to be logged in as the account holder...

<img width="1440" alt="Screenshot 2022-03-09 at 11 35 55" src="https://user-images.githubusercontent.com/60265851/157434467-c1cddb26-c3e0-442c-8dbb-9057e62d8ad7.png">

I really like WakaTime but until this is sorted I have removed it from Xcode and re-installed.

**Comment by alanhamlett** [comment]
<img width="258" alt="Screen Shot 2022-03-09 at 11 03 14 AM" src="https://user-images.githubusercontent.com/522344/157513111-8131803e-afdd-41fd-93f8-f39fdc3df17c.png">

**Comment by alanhamlett** [comment]
Apple's response:

> Thank you for contacting Apple Developer Technical Support (DTS). 
>
> The functionality you are looking to implement is not currently supported. We recommend that you file an enhancement request via [Feedback Assistant](https://feedbackassistant.apple.com/) (https://feedbackassistant.apple.com/).

**Comment by MohammadHossein-1996** [comment]
Is that mean you can not fix this issue?
